cms/src/components/content/EditContentForm.vue

```diff
@@ -350,6 +389,76 @@ const checkIfDirty = () => {
                         Only used for display, does not automatically publish at this date
                     </LInput>
+                    <LInput
+                        name="expiryDate"
+                        label="Expiry date"
+                        class="w-1/2"
+                        type="datetime-local"
+                        :disabled="!canTranslateContent"
+                    >
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            @click="() => calculateExpirationDate('1 Week')"
+                        >
+                            1W
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('2 Weeks')"
+                        >
+                            2W
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('1 Month')"
+                        >
+                            1M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('2 Months')"
+                        >
+                            2M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('3 Months')"
+                        >
+                            3M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('6 Months')"
+                        >
+                            6M
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            class="ml-1"
+                            @click="() => calculateExpirationDate('1 Year')"
+                        >
+                            1Y
+                        </LButton>
+                        <LButton
+                            type="button"
+                            variant="secondary"
+                            :icon="BackspaceIcon"
+                            class="ml-1 pt-2"
```

Comment:
The backspace button is not aligned with the other buttons:

<img width="114" alt="image" src="https://github.com/bccsa/luminary/assets/39056299/6ccadbda-939e-4207-9e8d-963fe5245ba7">

Probably something to do with the `pt-2` here that is not set on the other items.

Reply:
I have tried to set that class into the others items but it didn't work. I added the: inset-x-100 and bottom-0.2